client/index.js

```diff
@@ -81,3 +82,40 @@ const navigationRoot = document.getElementById(
 if ( navigationRoot ) {
 	render( <Navigation />, navigationRoot );
 }
+
+// Set up customer effort score survey.
+( function () {
+	if ( ! window.wcAdminFeatures[ 'customer-effort-score' ] ) {
+		return;
+	}
+
+	const root = appRoot || embeddedRoot;
+
+	function getJson() {
+		try {
+			return (
+				window.localStorage.getItem( 'customerEffortScoreTracks' ) ||
```

Comment:
I wasn't sure if there might have been an established pattern around localStorage use in WP core, but could only find [this persistence plugin](https://github.com/WordPress/gutenberg/tree/4e9b6b51caf315f7c71ee345a94793a35d83ac39/packages/data/src/plugins/persistence) for wp.data.

While I don't think we can just drop that in here and use, they do have an interesting fallback object store. @samueljseay any thoughts on localStorage use in wc-admin?